src/View/Helper/PrivateMessagesHelper.php

```diff
@@ -79,14 +79,12 @@ public function displayForm($pm, $recipients) {
 
         <div class="body">
             <div class="pmFields">
-            <div ng-hide="true">
             <?php
             if (!$isReply) {
-                echo $this->Form->input('messageId', array('value' => $pm->id));
+                echo $this->Form->hidden('messageId', array('value' => $pm->id));
             }
-            echo $this->Form->input('submitType', array('value' => ''));
+            echo $this->Form->hidden('submitType', array('value' => ''));
```

Comment:
Having the inputs as hidden lead to an error when saving a draft:

> Tampered field 'submitType' in POST data (expected value '' but found 'saveDraft') Cake\Controller\Exception\AuthSecurityException

Reply:
I solved the problem.